apps/src/studio/studio.js

```diff
@@ -3013,7 +3021,14 @@ Studio.displaySprite = function(i, isWalking) {
     spriteWalkIcon.setAttribute('visibility', 'hidden');
 
     xOffset = sprite.drawWidth * spriteFrameNumber(i);
-    yOffset = 0;
+
+    // For new versions of sprites (iceage/gumball), there are additional rows of
```

Comment:
we like to avoid mentioning specific skins in the code, and let the skin do that, but not such a big deal since this is a comment.

(i learned this the hard way when first doing frozen by having a lot of 'if skin == frozen' conditionals throughout the code, until @Bjvanminnen showed me a better path :)